docs/testing.rst

```diff
@@ -0,0 +1,45 @@
+=======
+Testing
+=======
+
+The Hy package provides a very basic pytest plugin out of the box to
+help testing Hy code.
+
+A simple test function with just four lines of code::
+
+    # content of test_sample.hy
```

Comment:
I suggest use semicolon for this line. Not a big deal, but I ran into this as I accidentally copied this line too.